hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/WritableFreezeStore.java

```diff
@@ -72,11 +72,11 @@ public Timestamp freezeTime() {
     /**
      * Sets or clears the update file hash.
      *
-     * @param updateFileHash The update file hash to set. If null, clears the update file hash.
+     * @param fileHash The update file hash to set. If null, clears the update file hash.
```

Comment:
Curious, why this change?

Reply:
Same as [here](https://github.com/hashgraph/hedera-services/pull/14685#discussion_r1711101452)